docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  web:
+    build:
+      context: .
+    depends_on:
+      - db
+    links:
+      - db
+    volumes:
+      - .:/app
+    ports:
+      - 4000:4000
+    environment:
+      - DATABASE=postgres
+      - DATABASE_USERNAME=postgres
+      - DATABASE_PASSWORD=mysecretpassword
+      - DB_HOST=db
+      - DB_DIALECT=postgres
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=mysecretpassword
+      - POSTGRES_DB=postgres
```

Comment:
These env vars do not seem to be referenced anywhere, are they automatically picked up by postgres?